lib/private/User/BasicAuthModule.php

```
@@ -66,6 +63,10 @@ public function getUserPassword(IRequest $request) {
 			return '';
 		}
 
+		if ($this->session->getSession()->exists('app_password')) {
+			// TODO: use proper password
+			throw new \Exception('Not implemented yet');
```

is this ever triggered ?

---

Not at the moment - this is a safety exception which will punch us in the face as soon as we use it accidentally.

Actually the whole method should not yet be called ... we could just throw the exception right away